osu.Framework/Graphics/UserInterface/TabControl.cs

```diff
@@ -224,10 +265,10 @@ private void removeTab(T value, bool removeFromDropdown = true)
         }
 
         /// <summary>
-        /// Adds an arbitrary <see cref="TabItem{T}"/> to the control.
+        /// Adds an arbitrary <see cref="TabItem{T}"/> to the <see cref="TabControl{T}"/>.
         /// </summary>
         /// <param name="tab">The tab to add.</param>
-        /// <param name="addToDropdown">Whether the tab should be added to the Dropdown if supported by the <see cref="TabControl{T}"/> implementation.</param>
+        /// <param name="addToDropdown">Whether the tab should be added to the dropdown if supported by the <see cref="TabControl{T}"/> implementation.</param>
```

Comment:
nit: maybe `cref` it?

```suggestion
        /// <param name="addToDropdown">Whether the tab should be added to the <see cref="Dropdown"/> if supported by the <see cref="TabControl{T}"/> implementation.</param>
```